src/librustc_target/abi/call/mod.rs

```diff
@@ -264,7 +264,8 @@ impl HomogeneousAggregate {
 }
 
 impl<'a, Ty> TyLayout<'a, Ty> {
-    fn is_aggregate(&self) -> bool {
+    /// Returns whether this layout is an aggregate.
+    pub fn is_aggregate(&self) -> bool {
```

Comment:
Ah I see. This is private for a reason: its meaning is specific to `abi::call`!
Should be named "`passes_like_aggregate`" or something.